BolajiOlajide/rmw-bot
# Created Migrations for Riders Model

#### What Does This PR Do?
Create `Users` Model

#### Description Of Task To Be Completed
To create necessary migrations for the User model.

#### Any Background Context You Want To Provide?

#### How can this be manually tested?
In terminal, after enabling the necessary environment, and in the root directory of the project, run `python manage.py db upgrade` This should create a `users` table in the db. If errors occur, ensure your `Postgres` DB is setup and included in the `.env` file. Sample at `.env.sample`

#### What are the relevant github issues (Link)?
Issue: #15 
#### Screenshot(s)
